9_ninth_hw/1_first_task.py

```diff
@@ -0,0 +1,108 @@
+"""
+Create a class describing any company. For example, Toshiba or Apple
+
+I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+I want to see clean code. I expect to see clean code with annotations.
+So far, no first-class connections with the second. In all methods, I expect to see docstrings with a sane description.
+"""
+from datetime import date, timedelta, datetime
+
+
+class Apple:
+    BRAND = "Apple"
+
+    def __init__(self, __website: str = "apple.com", __address: str = "silicone str"):
+        self.__website = __website
+        self.__address = __address
+        self.employees = 0
+
+    def new_employee(self, new_employees_num: int):
+        """
+        :param new_employees_num: update number of employees by the new employees
+        :return: new number of employees
+        """
+        self.employees += new_employees_num
+        return self.employees
+
+    def fire_employee(self, fire_employees_num: int):
+        """
+        :param fire_employees_num: update number of employees by reduction of employees amount
+        :return: new number of employees
+        """
+        self.employees -= fire_employees_num
+        return self.employees
+
+    def website(self):
+        """
+        :return: website name
+        """
+        return self.__website
+
+    def address(self):
+        """
+        :return: address name
+        """
+        return self.__address
```

Comment:
I guess website and address can be changed. SO please add setters.

Also use property for this getters